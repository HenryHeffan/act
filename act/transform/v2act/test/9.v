module blk0(e0, out);
  input [4:0] e0;
  output [4:0] out;
  wire f;
  assign out[4:0] = 5'b01x01;
endmodule
