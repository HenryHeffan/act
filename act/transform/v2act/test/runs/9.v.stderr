WARNING: File `9.vp', line 5, col 25
	Binary constant "01x01" has non-0/non-1 entries that were treated as zero
